R/utils.R

```diff
@@ -228,22 +217,27 @@ get_data_type_to_data_model_mapping <- function() {
 #' @keywords utils
 #' @export
 data_model.character <- function(x) {
-  # TODO: switch to gDRutils::get_experiment_groups()
-  # once we clean-up single-agent/combination assignemnts
-  exp_v <- get_data_type_to_data_model_mapping()
-  
-  checkmate::assert_subset(x, names(exp_v))
-  
-  exp_v[[x]]
+  checkmate::assert_subset(x, gDRutils::get_supported_experiments())
```

Comment:
@gladkia Please, check and confirm that it is what you meant (get rid of `get_data_type_to_data_model_mapping()` with refactor `data_model.character()` and `validate_data_models_availability()`

Reply:
@bczech can we assume that the data model is now simply the experiment name? I mean for `co-dilution` the data model will be the `co-dilution`. If this is the case the logic can be even more simplified.

Reply:
Continuation in [GDR-2542](https://jira.gene.com/jira/browse/GDR-2542)